great-idea-website/index.html

```diff
@@ -1,63 +1,94 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
 
   <title>Great Idea!</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Bangers%7CTitillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
-  <body>
+<body>
+  <nav>
+    <a href="">Services</a>
+    <a href="">Product</a>
+    <a href="">Vision</a>
+    <a href="">Features</a>
+    <a href="">About</a>
+    <a href="">Contact</a>
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </nav>
 
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
 
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
 
-  Get Started
 
-  <img src="img/header-img.png" alt="Image of a code snippet.">
 
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+  <header>
+    <div>
+      <h1 class= "Headerone">
+        Innovation
+        On
+        Demand
+      
+        Get Started
+      </h1>
+      <button class= "Button">
+          <img src="img/header-img.png" alt="Image of a code snippet.">
 
+      </button>
+    </div>
 
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
 
-  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+
+
+
+  </header>
   
-  Services
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+<section class="Section">
+
```

Comment:
I like that you used the `section` HTML tag.

Giving the section a class called `.Section` is redundant. Do you understand why? 

Keep away from using capital letters (like the 'S' in section), because they have a particular meaning when you move into javascript.